documentation/src/docs/asciidoc/release-notes/release-notes-5.12.0-RC2.adoc

```diff
@@ -48,7 +48,7 @@ repository on GitHub.
 * The `@TempDir` now warns when deleting symlinks that target locations outside the
   temporary directory to signal that the target file or directory is _not_ deleted, only
   the link to it.
-
+* New optional CLI options `--redirect-stdout` and `--redirect-stderr` to redirect stdout and stderr outputs to a file.
```

Comment:
At this point, this should go into 5.13 but I haven't created the release notes file, yet. Will do so today.

Reply:
Please rebase onto `main` and move this entry to the 5.13 release notes. Moreover, please hard-wrap it at 90 characters.